py/picca/delta_extraction/data_catalogues/desi_data.py

```diff
@@ -138,14 +140,9 @@ def __parse_config(self, config):
         DataError upon missing required variables
         """
         # instance variables
-        self.blinding = config.get("blinding")
-        if self.blinding is None:
-            raise DataError("Missing argument 'blinding' required by DesiData")
-        if self.blinding not in ACCEPTED_BLINDING_STRATEGIES:
-            raise DataError(
-                "Unrecognized blinding strategy. Accepted strategies "
-                f"are {ACCEPTED_BLINDING_STRATEGIES}. "
-                f"Found '{self.blinding}'")
+        self.unblind = config.getboolean("unblind")
+        if self.unblind is None:
+            raise DataError("Missing argument 'unblind' required by DesiData")
```

Comment:
@iprafols - Why do we need to specify unblind? I'd imagine it is ok to run with `unblind=None` for now, isn't it?

Reply:
When run with the scripts its value is loaded to False by default, the check is there in case the class is used in a notebook or similar to make sure that the default value is properly loaded when the variable is not specified